.github/workflows/ci.yml

```diff
@@ -68,7 +71,7 @@ jobs:
         ln -s .venv/lib/python*/site-packages/pydantic_settings pydantic_settings
         ln -s .venv/lib/python*/site-packages/pydantic_extra_types pydantic_extra_types
 
-    - run: pdm run mkdocs build
+    - run: pdm run mkdocs build -v
```

Comment:
Hoping to get some additional insights here if/when it fails. Hoping this matches the docs-update job closely enough now.

Reply:
If it passes, the next thing I'd try is to change the python version to 3.8 in the docs-update.yml job